src/pint/models/astrometry.py

```diff
@@ -10,6 +11,11 @@
 
 from loguru import logger as log
 
+try:
+    from erfa import ErfaWarning
+except ImportError:
+    from astropy._erfa import ErfaWarning
```

Comment:
Is this still relevant for our oldest acceptable version of astropy?

Reply:
I'm not sure.  I copied it from elsewhere just to be safe.  I guess I could try deleting it and see if the relevant tests pass.

Reply:
I'm not sure we have relevant tests for this. In any case if it's still littered throughout the code the right answer is, if it's unnecessary, a separate PR to get rid of it.